kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ClusterCondition.java

```diff
@@ -31,4 +38,16 @@ static ClusterCondition filterKindNotKnown(String cluster, String filterName, St
                 String.format("Filter \"%s\" has a kind %s that is not known to this operator.", filterName, kind));
     }
 
+    static ClusterCondition targetClusterRefNotExists(String cluster, TargetCluster targetCluster) {
```

Comment:
`ThingNotExists` is weird English (I know you're following the bad example of `filterNotExists`). Probably makes sense change both to `ThingNotFound`.